releasetool/data/roles/load_gpg_passphrase/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: get gpg passphrase
+  command: gopass show -c '{{ pass_name_gpg }}'
```

Comment:
Does the `pass` here represent GPG password? or go pass?

Reply:
`pass_name_gpg` is the "name of the (go)pass entry that contains the gpg passphrase"

Reply:
That was succintly confusing.